archinstall/locales/tr/LC_MESSAGES/base.po

```diff
@@ -0,0 +1,753 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
```

Comment:
You can put your name here so that you get credit

Reply:
Thanks for reminding. I edited the lines you specified and sent them again.